ledger-core/v2/application/builtin/contract/testwallet/testwallet.go

```diff
@@ -6,53 +6,54 @@
 package testwallet
 
 import (
-	"errors"
-	"fmt"
+       "errors"
```

Comment:
It's strange, but in that file all tabs were replaced with spaces. Why?

Reply:
hmmm...